src/OpenTelemetry/Trace/Processor/ExceptionProcessor.cs

```diff
@@ -62,8 +62,10 @@ public override void OnEnd(Activity activity)
 
         if (snapshot != pointers)
         {
+#pragma warning disable
             // TODO: Remove this when SetStatus is deprecated
             activity.SetStatus(Status.Error);
```

Comment:
could this be moved to the new SetStatus API?

Reply:
It already has a new SetStatus. As @vishweshbankwar mentioned, we need both ways.